sql/revenue_derived/client_ltv_v1/query.sql

```diff
@@ -209,65 +206,105 @@ with_caps AS (
     with_actions_per_day
   INNER JOIN
     cutoffs
+  USING
+    (engine)
+),
+total_engine_values AS (
+  SELECT
+    engine,
+    country,
+    SUM(ad_clicks) AS total_ad_clicks,
+    SUM(search_with_ads) AS total_searches_with_ads,
+    SUM(sap) AS total_sap,
+    SUM(tagged_sap) AS total_tagged_sap
+  FROM
+    engine_action_values
+  GROUP BY
+    engine,
+    country
+),
+future_values AS (
+  -- For each engine-country, this gives the expected future value of each action
+  SELECT
+    engine,
+    country,
+    SUM(revenue) AS revenue,
+    SUM(ad_clicks) AS total_ad_clicks,
+    SUM(ad_click_value * (ad_clicks / total_ad_clicks)) AS ad_click_value,
```

Comment:
Here's the algo we talked about (weighting value by number of ad clicks that month)